deploy/helms/legion/templates/edi/edi-rbac.yaml

```diff
@@ -16,10 +16,10 @@ rules:
   verbs:  ["list", "create", "patch", "get"]
 - apiGroups: ["apps"]
   resources: ["deployments"]
-  verbs:  ["list", "delete", "patch", "get"]
+  verbs:  ["list", "delete", "patch", "get", "create"]
```

Comment:
Why did you add new action?

Reply:
There are a lot of tests failing without this action: https://cc.epm.kharlamov.biz/jenkins/job/Deploy_Legion_Release/369/robot/report/log.html#s1-s1-t22-k2-k3
Not quite sure why this happened after version change, but this action solves the problem.

Reply:
Seems that now I understand why this helped.
Edi before was configured as `ReplicationController`, and now it is configured as `Deployment`. That is why now another rule is applied.
So we need get all actions from rules applied to `extensions` and add it to `apps`.

Reply:
It this is true, we have to remove RBAC rules for `ReplicationController`s, right? And update docs.

Reply:
If I understand everything correctly, we have to remove `deployments` from `apiGroups: ["extensions"]`.
Since Ingress is still configured through `extensions/v1beta1`, the rest of rule for `extensions` should remain the same.